packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -896,7 +931,8 @@ class DescopeWc extends BaseDescopeWc {
     const id = getFirstNonEmptyValue(formData, idFields);
     const password = getFirstNonEmptyValue(formData, passwordFields);
 
-    if (id && password) {
+    // PasswordCredential not supported in Firefox
+    if (id && password && globalThis.PasswordCredential) {
```

Comment:
We have the same check 2 lines below, can we remove one?

Reply:
Removed